source/_includes/sidebar.html

```diff
@@ -147,15 +147,9 @@
                 <ul>
                     <li><a href="{{ site.url }}/baikal/">Introduction</a></li>
                     <li><a href="{{ site.url }}/baikal/install/">Installation</a></li>
+                    <li><a href="{{ site.url }}/baikal/docker-install/">Installation using docker</a></li>
```

Comment:
```suggestion
                    <li><a href="{{ site.url }}/baikal/docker-install/">Installation using Docker</a></li>
```

Just a minor detail: upper case since Docker themselves use a capital "D" on their documentation.